lib/cmds/simulator.js

```diff
@@ -32,6 +32,25 @@ class Simulator {
       cmds.push("--mnemonic \"" + (simulatorMnemonic) +"\"");
     }
 
+    // the simulatorAccounts configuration overrides a mnemonic
+    let simulatorAccounts = this.blockchainConfig.simulatorAccounts || options.simulatorAccounts;
+    if (simulatorAccounts && simulatorAccounts.length > 0) {
+      simulatorAccounts.forEach((account, index) => {
```

Comment:
It would be better to use lib/utils/accountParser.js with its function `parseAccountsConfig` so that we enable the same functionalities as in the tests.
It takes care of the accounts AND of the balance, so you could remove a lot of the code here.
You would need to require Web3, but you can require it only in the `if` for `simulatorAccounts`.